.github/workflows/aws-staging.yml

```diff
@@ -17,7 +17,7 @@ on:
 
 env:
   AWS_REGION: us-east-1                                        # set this to your preferred AWS region, e.g. us-west-1
-  BUCKET_NAME: staging.docs.oak.tech
+  BUCKET_NAME: staging.docs.avaprotocol.org
```

Comment:
I don’t think the bucket name changed. They don’t need to change. Could you log on to AWS S3 and check it?

Reply:
Yes, It's not changed.